tests/integration/bundle/test_bundle.py

```diff
@@ -89,7 +89,7 @@ async def test_kill_pg_primary(ops_test: OpsTest):
     connstr = finos_databag.get("master", None)
     assert connstr, f"databag incorrectly populated, \n {finos_databag}"
 
-    pgb_unit = get_leader(ops_test, PGB).name
+    pgb_unit = await get_leader(ops_test, PGB)
```

Comment:
This helper function became asynchronous due to the change in an of its internal calls.